integration_test/scripts/publish-images.sh

```diff
@@ -32,16 +32,16 @@ publish_adapter () {
 if [ "$2" == "true" ]
 then
   # getting local tag version by filtering adapter image from list of images, getting tags
-  # and filtering version tag by `grep "\."` 
+  # and filtering version tag by `grep "\."`
   # have to do this because docker does not allow to search through tags in local storage
   version=`docker images | grep $base_name | awk '{print $2}' | grep "\." | head -n 1`
   version=${version[0]}
   import=$(publish_adapter import $version)
   export=$(publish_adapter export $version)
   body="$import\n$export"
   echo {\"tag_name\": \"$version\",\"name\": \"$version\",\"body\": \"$body\"} > /workspace/request.json
-  responseCode=$(curl -# -XPOST -H 'Content-Type:application/json' -H 'Accept:application/json' -w "%{http_code}" --data-binary @/workspace/request.json \
-  https://api.github.com/repos/$githubUser/$githubRepo/releases?access_token=$accessToken -o response.json)  
+  responseCode=$(curl -# -XPOST -H "Authorization: Bearer $accessToken" -H 'Content-Type:application/json' -H 'Accept:application/json' -w "%{http_code}" --data-binary @/workspace/request.json \
```

Comment:
might need to move the " before the $accessToken, not totally sure though

Reply:
Hmm I usually do `-H "Authorization: Bearer $(gcloud auth print-access-token)"` or `?access_token=$(gcloud auth print-access-token)` so I think it should be fine?

Reply:
Sg, let's give it a try